src/redis_cmd.cc

```diff
@@ -779,11 +779,13 @@ class CommandCAS : public Commander {
       std::string opt = Util::ToLower(args[i]);
       if (opt == "ex") {
         if (last_arg) return Status(Status::NotOK, errWrongNumOfArguments);
-        ttl_ = atoi(args_[++i].c_str());
+        auto parse_result = ParseInt<int>(args_[++i].c_str(), 10);
```

Comment:
It is expected to check the status, otherwise wrong user input will just terminate the kvrocks instance, which will cause serious security problems.